### PR TITLE
Feature/backend

### PR DESCRIPTION
**Pull request by zJakeLoo**
Implement backend in Go + GAE.
Separate backend/ and frontend/ to the specific folder.

### CODE REVIEW

**Comment by Daniel-Awai** [comment]
Looks clean, thanks for organizing the files. :shipit:

File: backend/backend.yaml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+application: asdvc-proj-845
+module: backend
+version: 1
+api_version: go1
+runtime: go
+
+handlers:
+- url: /.*
+  script: _go_app
```
Comment (dzmitry-tsurankou) [inline backend/backend.yaml]: newline is missing here

File: backend/club.go [hunk 2]
```diff
@@ -0,0 +1,25 @@
+package backend
+
+import (
+	"fmt"
```
Comment (dzmitry-tsurankou) [inline backend/club.go]: I understand it is go, but indentation sucks

File: backend/club.go [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+	http.HandleFunc("/api/clubs", getClubs)
+}
+
+func getClubs(w http.ResponseWriter, r *http.Request) {
+	fmt.Fprint(w, "Hello Weeoeoeoeo API")
+
+	// c := appengine.NewContext(r)
+}
+
+// Register club
```
Comment (dzmitry-tsurankou) [inline backend/club.go]: cad that commented code be removed?

File: backend/club.go (hunk 2 above)
Comment (zJakeLoo) [inline backend/club.go]: They have the 'go fmt' to do the formatting haha

File: backend/club.go (hunk 3 above)
Comment (zJakeLoo) [inline backend/club.go]: Removed in #2

**Pull request merged**